crates/swc_plugin_runner/src/lib.rs

```diff
@@ -7,8 +7,11 @@ use transform_executor::TransformExecutor;
 
 pub mod cache;
 mod host_environment;
+#[cfg(feature = "__rkyv")]
```

Comment:
I think it's better to have

```rust
#[cfg(all(not(feature = "rkyv"), not(feature = "rkyv-bytecheck-impl")))
compile_error!("Error message")
```

Reply:
`swc_plugin_*` is useless if there's no rkyv, and we should fail on compile time instead of runtime

Reply:
There's no fancy message, but this still fails on compile time if any plugin transform misses rkyv-impl or rkyv-bytecheck-impl.

Reply:
after all, I'm not too much worried about those case as `swc_core` should have right control for those, between `plugin_transform` vs `plugin_transform_bytecheck`.